src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
@MrTheodor, first of all, do not mix things in this PR. the precise idea is to remove (https://www.google.de/search?q=define:remove&ie=utf-8&oe=utf-8&gws_rd=cr&ei=BCoCWJWuLYr3aO3GuqAP) the FixedLists... so that a "newer" FixedList... can be created. The disconnect/connect+destructor will be my future PRs on this but this PR is only "remove"= cleanest approach. 
btw I believe there is no much discussion because of the "remove" term but I appreciate the feedback!